budgeteer-web-interface/application.properties

```diff
@@ -48,3 +48,31 @@ spring.datasource.password=
 #spring.datasource.platform=org.hibernate.dialect.Oracle10gDialect
 #spring.datasource.jndi-name=jdbc/budgeteer-db-mys
 #spring.datasource.platform=org.hibernate.dialect.MySQL57InnoDBDialect
+
+# Keycloak configuration
+# details about connecting budgeteer to keycloak are in the README
+adapter.keycloak.activated = false
+
```

Comment:
The application does not start, when no external `application.properties` is specified, as the `budgeteer-web-interface/application.properties` is not bundled inside the war. It must be placed under `budgeteer-web-interface/src/main/resources/application.properties` to serve as default fallback configuration.